i18n/translations/de.json

```diff
@@ -78,8 +78,8 @@
   "_inviteMoreToGroup.comment": "Room Info Panel: link to invite people to the group/room",
   "addServiceIntegration": "Service Integration hinzufügen",
   "_addServiceIntegration.comment": "Room Info Panel: link to add an integration to the current room",
-  "welcomeToPm": "Private Konversation mit {mate}",
```

Comment:
you actually can't update translations this way, you need to create a new key always.

Reply:
what you did will cause you issues later, because translation tool will have old keys

Reply:
@AleshaOleg is it correct that this is still an issue? Can you instead create a "welcomeToPrivateMessage" and a "privateMessageIntro" translation?

Reply:
@nikgraf, there is no issue. @kof thought I didn't make a pull request from lingohub. Everything ok